test/UserService.Validation.UnitTests/UserValidatorTests.cs

```diff
@@ -208,26 +211,67 @@ public void ShouldNotThrowValidationExceptionWhenDataIsValidAndContainsRussianSy
                 Email = "Example@gmail.com",
                 Status = "Example",
                 Password = "Example",
-                IsAdmin = false
+                IsAdmin = false,
+                Skills = new List<string>()
             };
 
             validator.TestValidate(request).ShouldNotHaveAnyValidationErrors();
         }
 
+        [Test]
         public void ShouldPassWhenDataIsValidWithoutMiddleName()
         {
             var request = new UserRequest
             {
                 Id = Guid.NewGuid(),
                 FirstName = "Example",
                 LastName = "Example",
+                Login = "admin",
                 Email = "Example@gmail.com",
                 Status = "Example",
                 Password = "Example",
-                IsAdmin = false
+                IsAdmin = false,
+                Skills = new List<string>()
             };
 
             validator.TestValidate(request).ShouldNotHaveAnyValidationErrors();
```

Comment:
You can use simple `validator.ShouldHaveValidationErrorFor(x => x, object);`

Reply:
I guess now i did what do you want

Reply:
fixed